app/pages/login/login.js

```diff
@@ -189,6 +190,10 @@ var Login = React.createClass({
     submit(formValues, function(err) {
       if (err) {
         var message = 'An error occured while logging in.';
+        if(err.status === 403){
+          self.props.onSubmitNotAuthorized();
+          return;
+        }
```

Comment:
This whole bit is a bit confusing to read. Since `onSubmitNotAuthorized` redirects to the sign-up verification page, the error message here will never get rendered. (a) Is that the intended behavior? (b) If yes, can you at least move the line defining the message below the `if` block where `onSubmitNotAuthorized` is called?